docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,448 @@
+# User Acquisition Analytics
+
+Organizations have until recently done most of their user acquisition analytics on structured data. Vector embeddings have changed this. Because vectors capture the semantic meaning and context of unstructured data such as text, they enable organizations to inform their user targeting strategies with more nuanced and detailed insight into what drives user behavior. But to use vectors to do nuanced user behavior analysis, you need to overcome several challenges - including computational complexity, interpretability, and finding the right embedding techniques.
+
+In this article, we'll show you how to use the Superlinked library to overcome these challenges - letting you leverage vector embedding power to **identify and analyze users on the basis of how they respond to particular ad messaging, target them more precisely, and improve conversion rates in your campaigns**.
+
+You can implement each step below as you read, in the corresponding [notebook](https://github.com/superlinked/superlinked/blob/main/notebook/analytics_user_acquisition.ipynb) and [colab](https://colab.research.google.com/github/superlinked/superlinked/blob/main/notebook/analytics_user_acquisition.ipynb).
+
+## Vector embedding - power & challenges
+
+By capturing intricate relationships and patterns between data points, and representing them as high-dimensional vectors in a latent space, embeddings empower you to extract deeper insights from complex datasets. With smart vectors, you can do more nuanced analysis, interpretation, and accurate predictions to inform your ad campaign decision-making.
+
+But while vector embeddings are a powerful tool for user analysis, they introduce **challenges**:
+
+- *Quality and relevance* - to achieve good retrieval results and avoid postprocessing and reranking, embedding generation techniques and parameters need to be selected carefully
```

Comment:
true but a bit unrelated - this article/notebook does not do retrieval, there are no queries

Reply:
@morkapronczay 
Would you remove:
- quality and relevance
- scalability and high-dimensional data
and just discuss 
- interpretability
?

or skip this little section altogether?